rialto_airflow/harvest/openalex.py

```diff
@@ -127,6 +127,7 @@ def normalize_publication(pub) -> dict:
     "apc_paid",
     "best_oa_location",
     "biblio",
+    "citation_normalized_percentile",
```

Comment:
This was a new column that is now coming back from OpenAlex and needed to be accounted for in the test.